### PR TITLE
add course and testing

### LINKED ISSUE #31: Create a POST endpoint for tutors to add a course.
Generate a course id using uuid. Retrieve tutor id from the JWT token.

### PR DESCRIPTION
**Pull request by txiong4**
Closes #31

### CODE REVIEW

File: routes/addCourse.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+const { v4 } = require("uuid");
+const express = require("express");
+const router = express.Router();
+const { Courses } = require("../db/Models");
+const jwtVerify = require("../helpers/jwtVerify");
+
+router.post("/", async(req, res) =>{
+
+  const token = req.signedCookies.user;
+  const decoded = await jwtVerify(token);
```
Comment (jimmoua) [inline routes/addCourse.js:10]: jwt verify can fail

File: tests/addCourse.test.js [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    expect(findCourse.course_name).toBe("Algebra");
+    expect(findCourse.initial_session_price).toBe(12.50);
+    expect(findCourse.session_hourly_rate).toBe(30.98);
+    expect(findCourse.tutor_id).toBe("txiong4");
+  });
+  test("course name is not valid", async() =>{
+    const res2 = await request(app)
+      .post("/addCourse")
+      .send({
+        course_name: null,
```
Comment (jimmoua) [inline tests/addCourse.test.js]: do not use `null` since it implies it was defined to be null somewhere.

**Review by jimmoua — CHANGES_REQUESTED** [review]

File: tests/addCourse.test.js (hunk 2 above)
Comment (txiong4) [reply tests/addCourse.test.js]: 3aab04731d99b1e2e232698545e4551eab407f5d

File: routes/addCourse.js (hunk 1 above)
Comment (txiong4) [reply routes/addCourse.js:10]: 3aab04731d99b1e2e232698545e4551eab407f5d

**Review by jimmoua — APPROVED** [review]

**Pull request merged**